examples/contrib/annotate-apply-time-mutations-simple/README.md

```diff
@@ -0,0 +1,32 @@
+# annotate-apply-time-mutations: Simple Example
+
+### Overview
+
+This example shows how the [annotate-apply-time-mutations] function works.
+
+Running `annotate-apply-time-mutations` function on the example package will:
+
+1.  Set `config.k8s.io/apply-time-mutation` annotation on resources with `apply-time-mutation` inline comments.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/annotate-apply-time-mutations-simple
```

Comment:
It should be `https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/contrib/annotate-apply-time-mutations-simple`